datalab/datalab_session/util.py

```diff
@@ -148,15 +156,26 @@ def get_hdu(basename: str, extension: str = 'SCI', source: str = 'archive') -> l
   
   return extension
 
+def fits_dimensions(fits_file) -> tuple:
```

Comment:
You're getting the image data dimensions right? I think you'd want to make this do something like:

```
def get_fits_dimensions(fits_file, extension: str = 'SCI'):
    return fits.open(fits_file)[extension].shape
```

Note how I changed the function name just to be a bit more descriptive